app/components/data/shared/cloud-file-picker/cloud-file-picker-dialog.html

```diff
@@ -1,15 +1,32 @@
 <bl-simple-form [submit]="submit" (done)="close()" [containerRef]="dialogRef" [multiUse]="false" actionName="Pick">
     <div *ngIf="container">
-        <bl-blob-files-browser #blobList [storageAccountId]="storageAccountId" [container]="container.id" [fileExplorerConfig]="fileExplorerConfig" [activeFile]="pickedFile"
+        <bl-blob-files-browser #blobList
+            [storageAccountId]="storageAccountId"
+            [container]="container.id"
+            [fileExplorerConfig]="fileExplorerConfig"
+            [activeFile]="pickedFile"
+            [filenameWildcards]="pickedFilter"
+            [recursiveFetch]="recursiveFetch"
```

Comment:
and here either recursive or loadAll

Reply:
yeah, always looked at this and never really liked the term. will change.